src/igvfd/types/file_set.py

```diff
@@ -288,6 +310,29 @@ def summary(self, request, assay_term, preferred_assay_title=None, readout=None,
                 sentence += phrase
         return sentence
 
+    @calculated_property(
+        condition='samples',
+        schema={
+            'title': 'Donors',
+            'description': 'The donors of the samples associated with this measurement set.',
```

Comment:
curated sets rather than measurment

Reply:
this is for measurement sets. curated sets do not calculate donors, the property is still submittable for them.